presto-main/src/test/java/com/facebook/presto/operator/TestClassificationMissRateAggregation.java

```diff
@@ -34,7 +34,7 @@ public TestClassificationMissRateAggregation()
         final ArrayList<Double> expected = new ArrayList<>();
         while (iterator.hasNext()) {
             final BucketResult result = iterator.next();
-            final double missRate = result.remainingFalseWeight / result.totalTrueWeight;
+            final double missRate = (result.totalTrueWeight - result.remainingTrueWeight) / result.totalTrueWeight;
             expected.add(missRate);
         }
         return expected;
```

Comment:
Did we only test execution path rather than real expected value? Ideally, unit test should exist to catch this error.

Reply:
I think it did test the expected value. But expected values are calculated at runtime, so the computation in unit test and function must keep consistent. Once the basic definition is wrong, we can't catch the error.